release-notes/README.md

```diff
@@ -4,6 +4,35 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## Sept 3, 2021
+
+### iOS AEPCore 3.3.0
+
+* Enables dispatch rules for event data forwarding.
```

Comment:
How about Added support for dispatch event rules consequence?

Reply:
We should add a separate page for this consequence (similar to attach/modify data) when we expose it in Launch.